pkg/syncthing/syncthing.go

```diff
@@ -878,6 +878,10 @@ func getParent(p *process.Process) (*process.Process, error) {
 		oktetoLog.Infof("parent name is the same: %s", name)
 		return getParent(parent)
 	}
+	if !strings.Contains(pName, "okteto") {
```

Comment:
We are getting parents until we get something different than `syncthing` which was supposed to be the `okteto` process. It seems that there are some OS that instead of syncthing being depending on okteto it depended on the okteto parent (which was the terminal) and was terminating the terminal process instead of okteto